lib/charms/prometheus_k8s/v0/prometheus.py

```diff
@@ -870,8 +870,7 @@ def _labeled_alert_groups(self):
             with path.open() as rule_file:
                 # Load a list of rules from file then add labels and filters
                 try:
-                    rules = yaml.safe_load(rule_file)
-                    rule = rules[0]  # each file is list of one rule
+                    rule = yaml.safe_load(rule_file)
```

Comment:
I am a bit worried about this. Because now, the Prometheus charm library (in particular the aggregator) will have to support two different formats for alerts rules - one for Bootstack team and one for every one else. This is of course possible and may be done with just a conditional here or there. If there is a more universal approach that will be ideal.

Reply:
No need to be concerned, the translation from machine charm alert rules to these will be in the LMAProxy